website/docs/pushdown.md

```diff
@@ -52,3 +52,23 @@ Numbers outside the range of the safe IEEE 754 precision (`< -9007199254740991.0
 will prefetch all numbers larger/smaller than max/min value of the range.
 
 <!-- markdownlint-restore -->
+
+## Supported pushdown on `find()` cursor methods
```

Comment:
That's a bit too specific for Node.js / JS driver. I think we should either document `find` command arguments or describe it in more general terms

Reply:
Let's just document for `find` command arguments, for now that's what we support.